pkg/tsdb/elasticsearch/time_series_query.go

```diff
@@ -242,9 +243,21 @@ func addTermsAgg(aggBuilder es.AggBuilder, bucketAgg *BucketAgg, metrics []*Metr
 		if orderBy, err := bucketAgg.Settings.Get("orderBy").String(); err == nil {
 			a.Order[orderBy] = bucketAgg.Settings.Get("order").MustString("desc")
 
-			if _, err := strconv.Atoi(orderBy); err == nil {
+			/*
+			   The format for extended stats and percentiles is {metricId}[bucket_path]
+			   for everything else it's just {metricId}
+			*/
+			metricIdRegex := regexp.MustCompile(`^(\d+)`)
+			metricIdMatches := metricIdRegex.FindStringSubmatch(orderBy)
```

Comment:
Is there any reason why this can't be simplified to `metricId := metricIdRegex.FindString(orderBy)`?

Reply:
Thanks for the suggesting! I will make the change and re-push.